docker/test/integration/features/python_with_modules.feature

```diff
@@ -1,30 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This use case is already covered by https://github.com/apache/nifi-minifi-cpp/blob/153c16f22f18bb8592ae0abe66e4acf411be18d9/docker/test/integration/features/python.feature#L90